### PR TITLE
[doc] add fullnode identity guide

### PR DESCRIPTION
**Pull request by sherry-x**
Our community is diving into the details of running fullnode, provide a bit more guide on setting up node identity. This is from our old operator runbook.

### CODE REVIEW

File: developer-docs-site/docs/tutorials/run-a-fullnode.md [hunk 1]
```diff
@@ -94,8 +94,83 @@ During the initial synchronization of your FullNode, there may be a lot of data
   du -cs -BM /opt/aptos/data
   ```
 
+## Advanced Guide
+
+If you want to dive into more customization of your node confg. This advanced guide will show you how to:
+* Create a static network identity for your new Fullnode
+* Retrieve the public network identity for other nodes allowlist
+* Start a node with or without a static network identity
+
```
Comment (davidiw) [inline developer-docs-site/docs/tutorials/run-a-fullnode.md:103]: White space

File: developer-docs-site/docs/tutorials/run-a-fullnode.md [hunk 2]
```diff
@@ -94,8 +94,83 @@ During the initial synchronization of your FullNode, there may be a lot of data
...
+
+### Create a static identity for a fullnode
+
+Fullnodes will automatically start up with a randomly generated network identity (a PeerId and a Public Key pair).  This works great for regular fullnodes, but if you need another node to allowlist you or provide specific permissions, or if you want to run your fullnode always with a same identity, creating a static network identity can help.
+
+1. Build the `aptos-operational-tool` using the [aptos-labs/aptos-core][] repo, we can build using cargo to run these tools. e.g.
+    ```
+    $ git clone https://github.com/aptos-labs/aptos-core.git
```
Comment (davidiw) [inline developer-docs-site/docs/tutorials/run-a-fullnode.md:111]: Do we have a docker equivalent? Also this feels like it's written where the user should already have our GitHub repo. But maybe whatever

**Review by davidiw — APPROVED** [review]
I love these kinda contributions! Folks need more homework!

File: developer-docs-site/docs/tutorials/run-a-fullnode.md (hunk 2 above)
Comment (sherry-x) [reply developer-docs-site/docs/tutorials/run-a-fullnode.md:110]: we do, I can put docker alternative here too

**Pull request merged**